web/locales/en/messages.ftl

```diff
@@ -0,0 +1,41 @@
+[[Layout]]
```

Comment:
In Fluent Syntax 0.5 sections like this one are replaced by _group comments_ starting with two `#`:

```properties
## Layout
speak = Speak
…
```